src/index.js

```diff
@@ -227,6 +227,27 @@ walletRouter.get('/address',
   res.send({ address });
 });
 
+/**
+ * GET request to get an address index
+ * For the docs, see api-docs.js
+ */
+walletRouter.get('/address-index',
+  query('address').isString(),
+  (req, res) => {
+  const validationResult = parametersValidation(req);
+  if (!validationResult.success) {
+    return res.status(400).json(validationResult);
+  }
+  const wallet = req.wallet;
+  const address = req.query.address;
+  const index = wallet.getAddressIndex(address);
+  if (index) {
```

Comment:
`0` is a valid index right? But I suspect this is a string so this isn't a problem.

Reply:
Yes, it's a valid index and is a problem because the return of `getAddressIndex` is an integer. I've updated the code to fix this. Can you see again please?

Reply:
Great!